packages/x-data-grid/src/models/events/gridEventLookup.ts

```diff
@@ -72,11 +72,12 @@ export interface GridRowEventLookup {
   };
   /**
    * Fired when the dragging of a row ends.
+   * It's mapped to the `dragend` DOM event.
    * @ignore - do not document.
    */
   rowDragEnd: {
     params: GridRowParams;
-    event: React.DragEvent<HTMLElement>;
+    event: DragEvent;
```

Comment:
Since this is undocumented, I feel it should be safe.
Even better, it's also a bugfix.

Opinions?

Reply:
Breaking changes only apply to our public API. This is not public, so it can be changed.